extensions/amp-sticky-ad/amp-sticky-ad.md

```diff
@@ -13,3 +13,52 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+
+# <a name="amp-sticky-ad"></a> `amp-sticky-ad`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>A stickyAd provides a way to fix ad at bottom of a page. The stickyAs serves as a container and the ad as its child will display as sticky-ad</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td><div><a href="https://www.ampproject.org/docs/reference/experimental.html">Experimental</a>; no validations yet.</div><div>Work in progress.</div></td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-sticky-ad" src="https://cdn.ampproject.org/v0/amp-sticky-ad-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
+    <td>NODISPLAY</td>
+  </tr>
+</table>
+
+## Behavior
+
+- There can be only one `<amp-sticky-ad>` in an AMP document. The `<amp-sticky-ad>` should only have one direct child of `<amp-ad>`.
+- The sticky will appear on the bottom of a page.
+- The height of the sticky-ad is whatever its child needs up to its max-height.
+- The max-height of the sticky-ad is 100px, if the height exceeds 100px then the height would be 100px and overflow content will be hidden.
+- The width of the sticky-ad is set to 100% ussing CSS and cannot be overridden.
+- The sticky-ad will display after scroll one viewport height from top provided there is at-least one more viewport of content available.
+- Swipe to dismiss (TBD)
+
+Example:
+```html
+<amp-sticky-ad layout="nodisplay">
```

Comment:
the indentation in this block is a bit messed up. I think `<amp-ad>` should only be indented 2 spaces and there's an extra space before the closing `</amp-sticky-ad>`.